packages/@aws-cdk/aws-elasticsearch/test/domain.test.ts

```diff
@@ -233,6 +233,23 @@ describe('log groups', () => {
     });
   });
 
+  test('appLogEnabled should create a uniquely named log group policy', () => {
```

Comment:
```suggestion
  test('log group policy is uniquely named for each domain', () => {
```

I would also add another domain to the test and make sure the policy names are different, the current assertion doesn't **strictly** validate that.